server/portal/settings/settings_custom.example.py

```diff
@@ -164,6 +168,13 @@
         'step': 'portal.apps.onboarding.steps.allocation.AllocationStep',
         'settings': {}
     },
+    {
+        'step': 'portal.apps.onboarding.steps.system_access.SystemAccessStep',
+        'settings': {
+            'required_systems': ['stampede2.tacc.utexas.edu','ls5.tacc.utexas.edu'],
+            'project_sql_id': 12345,
```

Comment:
What does this `project_sql_id` value do?

Reply:
Since the `SystemAccessStep` subclasses the `ProjectMembershipStep`, the functionality is much the same. There, `project_sql_id` is used to add a user to a predetermined TAS project that has the required system allocations needed for the portal.
https://github.com/TACC/Core-Portal/blob/main/server/portal/apps/onboarding/steps/project_membership.py#L37